packages/plugins/GeoLocation/src/store/actions.ts

```diff
@@ -128,6 +125,32 @@ const actions: PolarActionTree<GeoLocationState, GeoLocationGetters> = {
       dispatch('addMarker', transformedCoords)
     }
   },
+  printPositioningFailed(
```

Comment:
In `@polar/plugin-pins` this is simply done as part of the respective action.  
Why is it added here as a separate action?

Either this can be moved to `positioning` in this plugin or Pins should receive a similar function

Reply:
It has been pulled out here since the function became too long according to linting specifications. It also makes sense to me to pull it apart since the original function has become overly complex.

This is not the case for the Pins plugin, hence I've left it be.